src/backend/TrafficCourts/Staff.Service/Services/JJDisputeService.cs

```diff
@@ -105,4 +105,23 @@ public async Task<JJDispute> SubmitAdminResolutionAsync(string ticketNumber, boo
 
         return dispute;
     }
+
+    public async Task AssignJJDisputesToJJ(List<string> ticketNumbers, string? username, CancellationToken cancellationToken)
+    {
+        await GetOracleDataApi().AssignJJDisputesToJJAsync(ticketNumbers, username, cancellationToken);
```

Comment:
You will need to refactor this after my PR for TCVP-1778 is merged. You will not need to call `GetOracleDataApi()`. An `IOracleDataApiClient` instance is injected into the service.